tests/queries/test_batch_rewards.py

```diff
@@ -0,0 +1,49 @@
+import os
+import unittest
+
+import pandas.testing
+from dotenv import load_dotenv
+from pandas import DataFrame
+
+from src.pg_client import MultiInstanceDBFetcher
+from tests.db.pg_client import connect_and_populate_db_from
+
+
+class TestBatchRewards(unittest.TestCase):
+    def setUp(self) -> None:
+        db_url = "postgresql+psycopg2://postgres:postgres@localhost:5432/postgres"
+        self.fetcher = MultiInstanceDBFetcher([db_url])
```

Comment:
I assume this will be extended once the database is set up?

Reply:
Yea.. In the real setup we would provide both prod and barn urls to this structure. However, for testing we only have one DB.